### PR TITLE
Use kernel.logs_dir instead of kernel.root_dir in SimpleJobLauncher

### PR DESCRIPTION
**Pull request by xElysioN**
[//]: <> (<3 Thanks for taking the time to contribute! You're awesome! <3)

[//]: <> (If you've never contributed to this repository before, please read https://github.com/akeneo/pim-community-dev/blob/master/.github/CONTRIBUTING.md)

**Description (for Contributor and Core Developer)**

[//]: <> (What does this Pull Request do? reference the related issue?)
Fix issue #5212 
SimpleJobLauncher use a kernel.logs_dir instead of kernel.root_dir for logs

### CODE REVIEW

File: src/Akeneo/Bundle/BatchBundle/Launcher/SimpleJobLauncher.php [hunk 1]
```diff
@@ -80,7 +86,7 @@ public function launch(JobInstance $jobInstance, UserInterface $user, array $con
             escapeshellarg($jobInstance->getCode()),
             $executionId,
             !empty($configuration) ? sprintf('--config=%s', escapeshellarg($encodedConfiguration)) : '',
-            $this->rootDir
+            $this->getLogsDir()
```
Comment (aRn0D) [inline src/Akeneo/Bundle/BatchBundle/Launcher/SimpleJobLauncher.php]: Don't forget to change `%s/logs/batch_execute.log` in `%s/batch_execute.log`, we don't need to that anymore.

Comment (xElysioN) [inline src/Akeneo/Bundle/BatchBundle/Launcher/SimpleJobLauncher.php]: Yes, i forgot this part :p thx

File: src/Akeneo/Bundle/BatchBundle/Launcher/SimpleJobLauncher.php [hunk 2]
```diff
@@ -40,19 +43,22 @@ class SimpleJobLauncher implements JobLauncherInterface
...
+     * @param string                 $logsDir
      */
     public function __construct(
         JobRepositoryInterface $jobRepository,
         JobParametersFactory $jobParametersFactory,
         JobRegistry $jobRegistry,
         $rootDir,
-        $environment
+        $environment,
+        $logsDir
```
Comment (xElysioN) [inline src/Akeneo/Bundle/BatchBundle/Launcher/SimpleJobLauncher.php]: ok , i add it in the 1.7 changelog it's ok ?

Comment (a2xchip) [inline src/Akeneo/Bundle/BatchBundle/Launcher/SimpleJobLauncher.php]: No no @xElysioN add it to CHANGELOG.md or 1.8 if there is created cause you PR to master branch

Comment (a2xchip) [inline src/Akeneo/Bundle/BatchBundle/Launcher/SimpleJobLauncher.php]: exactly to `CHANGELOG-1.8.md`

File: CHANGELOG-1.7.md [hunk 3]
```diff
@@ -4,6 +4,12 @@
 
 - PIM-6207: correctly dismiss "Unsaved changes" message on system configuration
 
+## BC breaks
```
Comment (aRn0D) [inline CHANGELOG-1.7.md]: This needs to go in `CHANGELOG-1.8.md`

**Comment by aRn0D** [comment]
@xElysioN it seems that you need to update `spec\Akeneo\Bundle\BatchBundle\Launcher\SimpleJobLauncherSpec` :)

**Comment by xElysioN** [comment]
Update CHANGELOG-1.8.md with latest master version

**Comment by aRn0D** [comment]
@xElysioN you need to rebase your branch. I did not merge your PR before there was failure in `edit_family.feature`, could you have a look. Thank you

**Comment by aRn0D** [comment]
Don't worry I will rebase your branch to re-launcher the CI quickly! I think the failure has been fixed by another commit!

**Comment by aRn0D** [comment]
Ok CI re-launched!

**Comment by aRn0D** [comment]
@xElysioN ok merged!! Thank you for your contribution !!

**Comment by a2xchip** [comment]
:tada: Congrats @xElysioN

**Comment by xElysioN** [comment]
Oh, thank you :)

**Pull request merged**